dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -47,12 +47,29 @@ using namespace Microsoft::Windows::AppNotifications::Helpers;
 
 namespace winrt::Microsoft::Windows::AppNotifications::implementation
 {
+    // Must be static for AppNotificationManager::Deserialize
+
     winrt::Microsoft::Windows::AppNotifications::AppNotificationManager AppNotificationManager::Default()
     {
         static auto appNotificationManager{winrt::make<AppNotificationManager>()};
         return appNotificationManager;
     }
 
+    winrt::Windows::Foundation::IInspectable AppNotificationManager::Deserialize(winrt::Windows::Foundation::Uri const& /* uri */)
+    {
+        auto appNotificationManager{ Default() };
+        auto deserializer{ appNotificationManager.as<INotificationDeserializer>() };
```

Comment:
Please call it INotificationManagerDeserializer since this is NotificationManager